apps/hyperdrive-trading/src/ui/bridge/BridgeAssetsForm/BridgeAssetsActionButtons.tsx

```diff
@@ -91,12 +94,7 @@ export function BridgeAssetsActionButtons({
                   quote={quote}
                   balance={chainBalance}
                 />
-                <button
-                  type="button"
-                  className="daisy-btn daisy-btn-circle daisy-btn-primary w-32 disabled:bg-primary disabled:text-base-100 disabled:opacity-30"
-                >
-                  Bridge
-                </button>
+                {BridgeButton()}
```

Comment:
What's goin on here?
```suggestion
                <BridgeButton quote={quote} />
```